src/main/java/com/lpvs/util/LPVSWebhookUtil.java

```diff
@@ -109,13 +109,12 @@ public static boolean checkPayload(String payload) {
     }
 
     /**
-     * Retrieves the organization name from the repository URL in the LPVSQueue object.
+     * Checks if the given LPVSQueue object is not null and has a non-null repository URL.
      *
-     * @param webhookConfig LPVSQueue object containing repository information.
-     * @return The organization name.
-     * @throws IllegalArgumentException If the provided LPVSQueue object is null or if the repository URL is absent.
+     * @param webhookConfig the LPVSQueue object to check
+     * @throws IllegalArgumentException If the webhook configuration is null or if the repository URL is absent.
      */
-    public static String getRepositoryOrganization(LPVSQueue webhookConfig) {
+    private static void checkWebhookConfig(LPVSQueue webhookConfig) {
         if (null == webhookConfig) {
             log.error("Webhook Config is absent");
             throw new IllegalArgumentException("Webhook is absent");
```

Comment:
I think it is better to keep the same text of the error in log.error and in the IllegalArgumentException: "Webhook Config is absent"